### PR TITLE
[OLXEU-20712] - Add support for x-ray tracking

### PR DESCRIPTION
**Pull request by mbillewicz-olx**

### CODE REVIEW

File: dynamo/store.go [hunk 1]
```diff
@@ -58,6 +59,7 @@ type Store struct {
 	TableName   string
 	DynamoDB    *dynamodb.DynamoDB
 	writeBuffer []*dynamodb.WriteRequest
+	ctx         context.Context
```
Comment (zefhemel) [inline dynamo/store.go]: Errr, didn't we agree not to keep `ctx` in the Store struct?

File: lambda_util/gateway/ci.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: 0.2
```
Comment (zefhemel) [inline lambda_util/gateway/ci.yml]: I think we can get rid of this :-)

File: lambda_util/gateway/Readme.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<img src="http://tjholowaychuk.com:6000/svg/title/APEX/GATEWAY">
```
Comment (zefhemel) [inline lambda_util/gateway/Readme.md]: I think we can delete this file (but keep the license)

**Comment by zefhemel** [comment]
For the rest looks good, but let's not merge until we finish the app part.

File: dynamo/store.go (hunk 1 above)
Comment (mbillewicz-olx) [reply dynamo/store.go]: done

File: lambda_util/gateway/ci.yml (hunk 2 above)
Comment (mbillewicz-olx) [reply lambda_util/gateway/ci.yml]: done

File: lambda_util/gateway/Readme.md (hunk 3 above)
Comment (mbillewicz-olx) [reply lambda_util/gateway/Readme.md]: done

**Pull request merged**